src/test/java/com/baeldung/web/controller/BookControllerFindAllSummaryTest.java

```diff
@@ -98,8 +98,8 @@ public void testFindAllSummaryReturnsCorrectList() {
         when(repo.findAll()).thenReturn(bookList);
         List<BookResource> result = bookController.findAllSummary();
         assertEquals(2, result.size());
-        assertEquals("123", result.get(0).getIsbn());
-        assertEquals("456", result.get(1).getIsbn());
```

Comment:
method call was done on an object of B class, rather it should be done to the object of A class as the method belongs to A class: basically B has-a A relationship